README.md

```diff
@@ -1,25 +1,25 @@
 [![Build Status](https://github.com/google/vimscript-language-server/workflows/Rust/badge.svg)](https://github.com/google/vimscript-language-server/actions)
 
-# Vimscript Language Server
+# Vim script Language Server
 
-Implementation of Language Server protocol for vimscript / vimL language in
+Implementation of Language Server protocol for Vim script / vimL language in
 Rust.
 
 This project is still in very early development stage - it does not support all
-of vimscript syntax and most features are not implemented yet.
+of Vims cript syntax and most features are not implemented yet.
```

Comment:
@ujihisa Vims cript is incorrect.

Reply:
Omg great catch, thanks a lot! I amended this change now.